engine/security/trust/content_trust.md

```diff
@@ -121,222 +102,318 @@ to create new repositories, it is a good idea to store it offline in hardware.
-        },
-        “mode”: “disabled” | “permissive” | “enforced”,
-        “allow-expired-trust-cache”: true,
-    }
-}
+$ docker trust signer add --key cert.pem jeff dtr.example.com/admin/demo
+Adding signer "jeff" to dtr.example.com/admin/demo...
+Enter passphrase for new repository key with ID 10b5e94: 
 ```
 
-| **Stanza**                     | **Description**                        |
-|--------------------------------|----------------------------------------|
-| `trust-pinning:root-keys`      | Root key IDs are canonical IDs that sign the root metadata of the image trust data. In Docker Certified Trust (DCT), the root keys are unique certificates tying the name of the image to the repo metadata.  The private key ID (the canonical key ID) corresponding to the certificate does not depend on the image name. If an image’s name matches more than one glob, then the most specific (longest) one is chosen.            |
-| `trust-pinning:library-images` | This option pins the official libraries (`docker.io/library/*`) to the hard-coded Docker official images root key. DCT trusts the official images by default. This is in addition to whatever images are specified by `trust-pinning:root-keys`. If `trustpinning:root-keys` specifies a key mapping for `docker.io/library/*`, those keys will be preferred for trust pinning. Otherwise, if a more general `docker.io/*` or `*` are specified, the official images key will be preferred.                                                                |
-| `allow-expired-trust-cache`    | Specifies whether cached locally expired metadata validates images if an external server is unreachable or does not have image trust metadata. This is necessary for machines which may be often offline, as may be the case for edge. This does not provide mitigations against freeze attacks, which is a necessary to provide availability in low-connectivity environments.                                                |
-| `mode`                         | Specifies whether DCT is enabled and enforced. Valid modes are: `disabled`: Verification is not active and the remainder of the content-trust related metadata will be ignored. *NOTE* that this is the default configuration if `mode` is not specified. `permissive`: Verification will be performed, but only failures will only be logged and remain unenforced. This configuration is intended for testing of changes related to content-trust. `enforced`: DCT will be enforced and an image that cannot be verified successfully will not be pulled or run.  |
 
-***Note:*** The DCT configuration defined here is agnostic of any policy defined in 
-[UCP](https://docs.docker.com/v17.09/datacenter/ucp/2.0/guides/content-trust/#configure-ucp). 
-Images that can be deployed by the UCP trust policy but are disallowed by the Docker Engine 
-configuration will not successfully be deployed or run on that engine.
+Finally we will use the delegation private key to sign a particular tag and 
```

Comment:
minor - "delegation's private key" (and public key) is used elsewhere.  Probably want to be consistent between "delegation" or "delegation"

Reply:
Just for clarification, should I be referring to delegations key's as public and private?

Reply:
Ah ok, no problem. I will keep it inline with existing documentation and make it no possessive :)